js/main.js

```diff
@@ -20,6 +20,83 @@ function save(event) {
   data.entries.unshift(list);
   $img.setAttribute('src', '/images/placeholder-image-square.jpg');
   $journal.reset();
+  $uList.prepend(renderEntry(list));
+  viewSwap('entries');
+  if ($noEntry.className === 'column-full no-entry') {
+    toggleNoEntries();
+  }
 }
 
 $journal.addEventListener('submit', save);
+
+function renderEntry(entry) {
+  var $entryList = document.createElement('li');
+  document.querySelector('ul').appendChild($entryList);
```

Comment:
You shouldn't be appending to the `ul` here, this function should only be responsible for creating the DOM tree for this entry. The actual appending to the "entry-list" should happen outside of this function, wherever you call it